test/default_target_proposal_test.rb

```diff
@@ -11,4 +11,52 @@
       expect(subject.new.call(["Description"])).to be_a(::Hash)
     end
   end
+
+  describe "#call with MakeProposal command" do
+    before do
+      allow(Yast::ServicesManagerTarget).to receive(:read) # avoid real read
+      allow(Yast::Stage).to receive(:initial).and_return(true) # skip state validation
+    end
+
+    context "auto mode" do
+      before do
+        allow(Yast::Mode).to receive(:autoinst).and_return(true)
+      end
+
+      # TODO how to test it?
+      it "keep default target as it is before"
```

Comment:
is -> was

Reply:
What about mocking `ServicesManagerTarget.default_target` to some value and checking that it is [assigned](https://github.com/yast/yast-services-manager/blob/ef8b472b777e0f4e9b2d59ae7de46b0b741c0920/src/lib/services-manager/clients/default_target_proposal.rb#L187)?

Reply:
@imobachgs checking that it is assigned is quite hard, as it is accessible only as localized output of proposal output. It is not public reader.